Makefile.am

```diff
@@ -25,13 +25,15 @@ libyaksa_la_LDFLAGS = -version-info @libyaksa_so_version@
 
 include $(top_srcdir)/src/Makefile.mk
 
+test_deps = libyaksa.la test/dtpools/libdtpools.la
```

Comment:
Shouldn't `test/dtpools/libdtpools.la` be added to `test_deps` inside `test/dtpools/Makefile.mk`?  That's the Makefile that's generating this library.

Reply:
Sounds good.

Reply:
done.

Reply:
Also, `test_deps` should be a dependency for the test files themselves, not for `make testing`.  Otherwise, when someone builds the tests, the dependency won't reflect correctly.

Reply:
That means we need spell out the dependency for each test program... why doesn't `automake` do that for me (since it's in `LDADD`). Let me first check see if I missed anything...

Reply:
It turns out we can remove this `test_deps`. Each test program already has them as dependencies.